wezterm-gui/src/selection.rs

```diff
@@ -100,7 +224,7 @@ impl SelectionRange {
                 Ordering::Greater => return Ordering::Greater,
                 // If the zone starts on the same line then check that the
                 // x position is within bounds
-                Ordering::Equal => match zone.start_x.cmp(&start.x) {
+                Ordering::Equal => match zone.start_x.partial_cmp(&start.x).unwrap() {
```

Comment:
The unwrap here feels dangerous; can you adjust this match block to deal with the `None` case, or just use `cmp`?

Reply:
I used something similar to `cmp`. Is it fine like this?